warp10/src/main/java/io/warp10/continuum/ingress/Ingress.java

```diff
@@ -1622,6 +1622,10 @@ public void handleDelete(String target, Request baseRequest, HttpServletRequest
 
       String selector = request.getParameter(Constants.HTTP_PARAM_SELECTOR);
 
+      if (null == selector) {
+          throw new IOException("Invalid selector query parameter, should not be empty.");
```

Comment:
Could you rather use resp.sendError and use the constant defined in class `Constants` for the parameter name.

Also ensure the same test is performed in the standalone delete endpoint for coherency.

Reply:
Sure, I made the changes.